client-js/app/elements/labrad-app.ts

```diff
@@ -32,8 +60,12 @@ export class LabradApp extends polymer.Base {
   }
 
   doLogout(event) {
-    window.localStorage.removeItem('labrad-credentials');
-    window.sessionStorage.removeItem('labrad-credentials');
+    var key = 'labrad-credentials';
+    if (this.host) {
+      key += '.' + this.host;
+    }
+    window.localStorage.removeItem(key);
+    window.sessionStorage.removeItem(key);
```

Comment:
What's going on here? Are you storing separate session keys for being connected to multiple hosts?